share/shutter/resources/modules/Shutter/App/Directories.pm

```diff
@@ -66,7 +65,7 @@ sub get_home_dir   {Glib::get_home_dir}
 sub get_config_dir {Glib::get_user_config_dir}
 
 sub create_hidden_home_dir_if_not_exist {
```

Comment:
This function name doesn't reflect anymore what this function does, as the shutter dir is not longer "hidden", and not necessary in home